.github/workflows/check-python-dependencies.yml

```diff
@@ -1,6 +1,7 @@
 name: Check Python dependencies
 on:
   pull_request:
+    branches: [ master ]
```

Comment:
These changes were necessary to avoid PRs created from the repository itself (and not forks, like dependabot's) run CI jobs twice (they were triggering both `push` and `pull_request`)